ush/config.sh_rrfs_a_enkf_n3

```diff
@@ -0,0 +1,178 @@
+MACHINE="wcoss2"
+version="v0.3.7"
+ACCOUNT="RRFS_DEV"
+#RESERVATION="rrfsdet"
+EXPT_BASEDIR="/lfs/h2/emc/lam/noscrub/emc.lam/rrfs/${version}"
+EXPT_SUBDIR="rrfs_na_ens"
+
+PREDEF_GRID_NAME="RRFS_NA_3km"
+. set_rrfs_config_general.sh
+
+DO_ENSEMBLE="TRUE"
+#DO_ENSFCST="TRUE"
+#DO_DACYCLE="TRUE"
+#DO_SURFACE_CYCLE="TRUE"
+DO_SPINUP="TRUE"
+DO_SAVE_INPUT="TRUE"
+DO_POST_SPINUP="FALSE"
+DO_POST_PROD="FALSE"
+#DO_RETRO="FALSE"
+DO_NONVAR_CLDANAL="TRUE"
+#DO_ENVAR_RADAR_REF="FALSE"
+DO_SMOKE_DUST="FALSE"
+#DO_REFL2TTEN="FALSE"
+#RADARREFL_TIMELEVEL=(0)
+#FH_DFI_RADAR="0.0,0.25,0.5"
+#DO_SOIL_ADJUST="TRUE"
+#DO_RADDA="FALSE"
+#DO_BUFRSND="TRUE"
+#USE_FVCOM="TRUE"
+#PREP_FVCOM="TREU"
+
+if [[ ${DO_ENSFCST} == "TRUE" ]] ; then
+  DO_SPINUP="FALSE"
+  DO_SAVE_INPUT="FALSE"
+  DO_NONVAR_CLDANAL="FALSE"
+  DO_POST_PROD="TRUE"
+fi
+
+NET="rrfs_a"
+TAG="nenkf37"
+ARCHIVEDIR="/NCEPDEV/emc-meso/1year/emc.lam"
+NCL_REGION="conus"
+MODEL="RRFS_A"
+RUN="rrfs_a"
+envir="para"
+
+EXTRN_MDL_ICS_OFFSET_HRS="30"
+LBC_SPEC_INTVL_HRS="1"
+EXTRN_MDL_LBCS_OFFSET_HRS="6"
+BOUNDARY_LEN_HRS="12"
+BOUNDARY_PROC_GROUP_NUM="12"
+
+# avaialble retro period:
+# 20210511-20210531; 20210718-20210801
+DATE_FIRST_CYCL="20230202"
+DATE_LAST_CYCL="20230331"
+CYCL_HRS=( "00" "12" )
+CYCL_HRS_SPINSTART=("06" "18")
+CYCL_HRS_PRODSTART=("07" "19")
+#CYCL_HRS_STOCH=("00" "12")
+#CYCL_HRS_RECENTER=("19")
+CYCLEMONTH="02,03"
+CYCLEDAY="*"
+
+STARTYEAR=${DATE_FIRST_CYCL:0:4}
+STARTMONTH=${DATE_FIRST_CYCL:4:2}
+STARTDAY=${DATE_FIRST_CYCL:6:2}
+STARTHOUR="00"
+ENDYEAR=${DATE_LAST_CYCL:0:4}
+ENDMONTH=${DATE_LAST_CYCL:4:2}
+ENDDAY=${DATE_LAST_CYCL:6:2}
+ENDHOUR="23"
+
+PREEXISTING_DIR_METHOD="upgrade"
+INITIAL_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 12:00:00"
+BOUNDARY_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 06:00:00"
+PROD_CYCLEDEF="00 01-05,07-11,13-17,19-23 ${CYCLEDAY} ${CYCLEMONTH} ${STARTYEAR} *"
+PRODLONG_CYCLEDEF="${DATE_FIRST_CYCL}0000 ${DATE_LAST_CYCL}2300 06:00:00"
+#RECENTER_CYCLEDEF="00 19 * 10 2022 *"
+ARCHIVE_CYCLEDEF="${DATE_FIRST_CYCL}1400 ${DATE_LAST_CYCL}2300 24:00:00"
+if [[ ${DO_ENSFCST} == "TRUE" ]] ; then
+  BOUNDARY_LEN_HRS="36"
+  DO_SPINUP="FALSE"
+  INITIAL_CYCLEDEF="00 01 01 01 2100 *"
+  BOUNDARY_CYCLEDEF="${DATE_FIRST_CYCL}1200 ${DATE_LAST_CYCL}2300 12:00:00"
+  PROD_CYCLEDEF="${DATE_FIRST_CYCL}1200 ${DATE_LAST_CYCL}2300 12:00:00"
+  PRODLONG_CYCLEDEF="00 01 01 01 2100 *"
+fi
+if [[ $DO_SPINUP == "TRUE" ]] ; then
+  SPINUP_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 12:00:00"
+fi
+if [[ $DO_SAVE_INPUT == "TRUE" ]] ; then
```

Comment:
Please update this to use "DO_SAVE_DA_OUTPUT" and "SAVEDA_CYCLEDEF"

Reply:
Done